engine/robot_logic/traversal.py

```diff
@@ -64,17 +68,34 @@ def move_to_target_node(robot_state, target, allowed_dist_error, database):
     robot_state.goal_location = target
     robot_state.loc_pid_x.reset_integral()
     robot_state.loc_pid_y.reset_integral()
+    using_heading_pid = True
+    simulate = False
+    if simulate:
+        i = 0
+    import matplotlib.pyplot as plt
```

Comment:
why is the import statement here?

Reply:
it's not needed elsewhere. If we're only using it in move_to_target_node, I dunno if we need to import it globally